enterprise/frontend/src/embedding-sdk/components/private/InteractiveAdHocQuestion.tsx

```diff
@@ -63,7 +63,7 @@ export const InteractiveAdHocQuestion = ({
       componentPlugins={plugins}
       onBeforeSave={onBeforeSave}
       onSave={onSave}
-      entityTypeFilter={entityTypeFilter}
```

Comment:
we agreed on Slack that we should change the prop name to `entityTypes` to match what we have for interactive embedding `entity_types`